lib/get-all-tweets.js

```diff
@@ -1,15 +1,61 @@
 const Twitter = require('twitter');
 
-module.exports = function () {
-	let client = new Twitter({
+module.exports = function getAllTweets() {
+	// function writeTweetsToFile(tweets, filepath = 'tmp/all-tweets.json') {
+	// 	const fs = require('fs');
+	// 	fs.writeFile(filepath, JSON.stringify(tweets), (err) => {
+	// 		if (err) {
+	// 			console.log(err);
+	// 			return err;
+	// 		}
+	// 	});
+	// }
+
+	function saveToDynamoDB(tweets) {
+		const AWS = require('aws-sdk');
+		// AWS.config.update({
+		//   region: "us-east-1",
+		//   endpoint: "http://localhost:8000"
+		// });
+		AWS.config.update({
+			region: 'us-east-1',
+		});
+
+		const docClient = new AWS.DynamoDB.DocumentClient();
+		tweets.forEach((tweet) => {
```

Comment:
instead of a `for` loop, I moved this `forEach`. I need to research on lambda or aws sdk a bit more because if they do promises for docClient we can just refactor